lib/redux/onboarding/identity_verification/onboarding_identity_verification_action.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:solarisdemo/models/onboarding/onboarding_identity_verification_error_type.dart';
+
```

Comment:
I would remove the "url for integration" part and have something like `CreateIdentificationCommandAction`, `CreateIdentificationSuccessEventAction`